Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java

```diff
@@ -2169,6 +2170,256 @@ public void test_showCardInfo() {
         });
     }
 
+    private static final Map<String, Set<Integer>> intentionalUnimplemented = new HashMap<>();
+
+    static {
+        intentionalUnimplemented.put(
+                "LEA", new HashSet<>(Arrays.asList(
+                        63, 78,             // Text change
+                        96, 99, 102         // Ante
```

Comment:
1. Old set's cards can change their unofficial card numbers due scryfall reorganization;
2. Card names are more useful and readable, also you don't need to store it per set (many sets can contains same card);

Reply:
In my mind this is not for comparing to Scryfall or mtgjson data.
This is only using the ExpansionSet data, so there is no way to know what a missing card name should have as Card Number.

Basically for LEA, we are missing card numbers 63, 78, 96, 99, 102 and 249, and out of those, only 249 (Illusionary Mask) makes sense to implement as Ante and Text Change are for now considered not possible to implement.

This intentionalUnimplemented data will not change much in the future. Apart from the occasional preconstructed list having a hole due to card not being printed in the end.

Reply:
Yeah, incomplete sets and unimplemented cards aren't quite the same thing. I'd be surprised if scryfall reorganized collector numbers for old sets again. From what I can tell this is a sensible way to do it.